SQL/updates/12-13.sql

```diff
@@ -0,0 +1,4 @@
+#Updating the SQL from version 12 to version 13. -Kyep
+#Removes support for the "Android" alt title for the Cyborg job
+
+UPDATE `characters` SET player_alt_titles = REPLACE(player_alt_titles, 'Cyborg=Android','Cyborg=Robot') WHERE player_alt_titles LIKE '%Cyborg=Android%';
```

Comment:
DB version is only really incremented when the database undergoes a schema edit (table modification). The game can likely gracefully handle this, and a DB version bump should not be done for the sake of an update query imo

Reply:
I'd rather not update the DB version for this either, but Fox requested it.

Reply:
I didn't request a DB version update; I only requested a change be made to cleanse the existing table---not update the entire schema.

Reply:
Okay, PR updated.